### PR TITLE
DTSBPS-687: fortify integration on nightly build

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
DTSBPS-687

### Change description ###
Fortify integration on nightly build

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mdayican — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java [hunk 1]
```diff
@@ -19,6 +20,7 @@
 import java.util.zip.ZipInputStream;
 
 import static com.google.common.io.ByteStreams.toByteArray;
+import static org.apache.commons.io.FileUtils.*;
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java:23]: no *

**Review by mdayican — CHANGES_REQUESTED** [review]
why did you change `ZipFileProcessor.java` ?

**Comment by apereverzin** [comment]
Yes, we should have only relevant changes in PRs. Adding static import should go to a separate PR.

**Review by mdayican — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java (hunk 1 above)
Comment (rishikrsharma) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java]: reverted.

**Comment by rishikrsharma** [comment]
> why did you change `ZipFileProcessor.java` ?

**Pull request merged**